app/javascript/components/rooms/RoomsList.jsx

```diff
@@ -36,12 +33,9 @@ export default function RoomsList() {
       </Stack>
       <Row md={4} className="g-4 pb-4 mt-4">
         {
-          rooms.sort((a, b) => b.online - a.online).filter((room) => {
-            if (room.name.toLowerCase().includes(search.toLowerCase())) {
-              return room;
-            }
-            return false;
-          }).map((room) => (
+          // eslint-disable-next-line react/no-array-index-key
+          (isLoading && [...Array(9)].map((val, idx) => <Col key={idx} className="mt-0 mb-4"><RoomPlaceHolder /></Col>))
```

Comment:
This loading feedback felt more adequate in that area and offers a better UX,
It will only be shown for the first fetch, afterwards previous data will be guarded/kept until new data comes and seamlessly replace it,
![image](https://user-images.githubusercontent.com/29759616/193340931-1045874f-09b3-4cfa-ac46-4130191b97f0.png)

Reply:
Looks promising! Eager to get rid of the spinners

Reply:
Sure,